lib/fast_jsonapi/object_serializer.rb

```diff
@@ -105,6 +113,12 @@ def deep_symbolize(collection)
       end
     end
 
+    def serializer_for(record)
```

Comment:
This needs to play well with the existing method:
https://github.com/jsonapi-serializer/jsonapi-serializer/blob/master/lib/fast_jsonapi/object_serializer.rb#L303

Reply:
Crap, sorry! Missed that (and so did the tests I guess? haha)